File: SE4-PY/ejercicio2.py
an = int(input("Año: "))

if ((an % 4 == 0 and an % 100 != 0) or an % 400 == 0):
    print("\nBisiesto")
else: print("\nNo bisiesto")

if (an%2 == 0): print("Par")
else: print("Impar")


    

File: SE4-C/ejercicio2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SE4_C
{
    internal class ejercicio2
    {
        int an;

        Console.Write("Anio :");

      an = int.Parse(Console.ReadLine());



      if ((an % 4 == 00 && an % 100 != 0) || an % 400 == 0)

      {

        Console.WriteLine("\nEs bisiesto.");

      }

      else

      {

        Console.WriteLine("\nNo es bisiesto.");

      }



if (an % 2 == 0)

{

    Console.WriteLine("\nEs par");

}

else

{

    Console.WriteLine("\nEs impar");

}
    }
}
